README.md

```diff
@@ -151,14 +170,15 @@ Use `make` to build and test the CLI. Makefile help can be found:
 
 ```
 $ make help
-build             Build program binary for current os/arch
-build-all         Build all targets
-build-linux       Build program binary for linux x86_64
-build-darwin      Build program binary for darwin x86_64
-build-windows     Build program binary for windows x86_64
-test              Run tests
-fmt               Run gofmt on all source files
-clean             Cleanup everything
+build                Build program binary for current os/arch
+doc                  Build documentation (markdown)
+build-all            Build all targets
+build-linux          Build program binary for linux x86_64
+build-darwin         Build program binary for darwin x86_64
```

Comment:
we're supporting darwin/x86_64 and darwin/arm64 now, correct? If so, these should probably reflect it. 

(Although; that does raise another question - what about, say, linux/arm64?)

Reply:
yeah moving to another PR